### PR TITLE
Add @RyanL1997 to maintainers

### PR DESCRIPTION
**Pull request by cliu123**
Signed-off-by: Chang Liu <lc12251109@gmail.com>

### Description
Adding @RyanL1997 to the maintainers of this repo

[Nomination, vote and interest](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer) have already happened in private email threads with all current maintainers and @RyanL1997 .

### Category
Maintenance

### Why these changes are required?
@RyanL1997 has been actively making contributions.
Screen Shot 2022-10-05 at 4 40 19 PM

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -16,4 +17,4 @@
 | --------------- | --------- | ----------- |
 | Yan Zeng         | [zengyan-amazon](https://github.com/zengyan-amazon)   | Amazon      |
 | Bandini Bhopi    | [bandinib-amzn](https://github.com/bandinib-amzn)     | Amazon      |
-| Tianle Huang     | [tianleh](https://github.com/tianleh)                 | Amazon      |
```
Comment (davidlago) [inline MAINTAINERS.md:19]: I've been squinting at this line for a minute but can't quite see the difference. Maybe a newline or whitespace character somewhere?

**Review by davidlago — APPROVED** [review]
The current maintainer team has reviewed @RyanL1997's contributions and has voted unanimously to offer him maintainership. Welcome to the team, Ryan!

File: MAINTAINERS.md (hunk 1 above)
Comment (cliu123) [reply MAINTAINERS.md:19]: It is a line number change. It looks like that Github diff only shows the line number change the last line.

**Review by cwperks — APPROVED** [review]

**Comment by cliu123** [comment]
CI fails because 3.0 build fails. Merging this as it is a documentation only changes.

**Review by DarshitChanpura — APPROVED** [review]
Welcome aboard @RyanL1997 !

**Review by peternied — APPROVED** [review]

**Pull request merged**